chapters/classes.tex

```diff
@@ -694,13 +690,9 @@ \subsection{Short Class Definitions}\label{short-class-definitions}
 \end{lstlisting}
 \end{example}
 
-If a short class definition does not specify any specialized class the
-new class definition will inherit the specialized class (this rule
-applies iteratively and also for redeclare).
+If a short class definition does not specify any specialized class the new class definition will inherit the specialized class (this rule applies iteratively and also for redeclare).
 
-A base-prefix applied in the short-class definition does not influence
-its type, but is applied to components declared of this type or types
-derived from it; see also \cref{restriction-on-combining-base-classes-and-other-elements}.
+A base-prefix applied in the short class definition does not influence its type, but is applied to components declared of this type or types derived from it; see also \cref{restriction-on-combining-base-classes-and-other-elements}.
```

Comment:
I believe that both base-prefix and short-class-definition should refer to the grammar constructs; so change from short-class definition to short-class-definition?

Reply:
I'll try not to think too much (I know I have a general tendency to think less in terms references to the grammar constructs, and more in terms of references to the corresponding language concepts).  (In this case `base-prefix` was already turned into a grammar construct reference.)  Fixed.  Please resolve if good.